musicbot/bot.py

```diff
@@ -1702,6 +1702,47 @@ def check(m):
                 delete_after=20
             )
 
+    async def cmd_remove(self, message, player, index=None):
+        """
+        Usage:
+            {command_prefix}remove [number]
+	    
+        Removes a song from the queue at the given position, where the position is a number from {command_prefix}queue.
+        """
+
+        # if the queue has as many entries as the number they
+        # input, we know there's a song there
+        # if the user inputs 0, tell them to use skip instead
+        if not index:
+            raise exceptions.CommandError("No input found. Try again with a position from {command_prefix}queue.", expire_in=20)
+
+        try:
+            index = int(index)
+
+        except ValueError:
+            raise exceptions.CommandError('{} is not a valid number.'.format(index), expire_in=20)
+
+        if index <= len(player.playlist.entries):
+            if index > 0:
+                song_title = player.playlist.entries[index-1].title
+                
+                try:
+                    player.playlist.remove_entry((index)-1)
+
+                # this catches the rare case where the player
+                # loads a song and we try to remove a position on
+                # the very end of the queue that no longer exists
+                except IndexError:
+                    raise exceptions.CommandError("Something went wrong. Please try again.", expire_in=20)
+                
+                return Response(':white_check_mark: **' + song_title + '** removed.', delete_after=20)
```

Comment:
Copy the way we embed emojis like that in strings: https://github.com/SexualRhinoceros/MusicBot/blob/review/musicbot/bot.py#L1035